### PR TITLE
Fix condition logic for multiple selection controls in LBF

### PR DESCRIPTION
**Pull request by matrix-olegsv**
This fix allows to set conditions to show or hide other controls with several values  selected in multiple SELECT control.

### CODE REVIEW

File: library/options.inc.php [hunk 1]
```diff
@@ -85,9 +85,9 @@ function generate_select_list($tag_name, $list_id, $currvalue, $title, $empty_na
 	
 	$tag_name_esc = attr($tag_name);
 	
-	if ($multiple) {
+	/*if ($multiple) {
 		$tag_name_esc = $tag_name_esc . "[]";
-	}
+	}*/
```
Comment (bradymiller) [inline library/options.inc.php]: Remove code rather than comment it out.

File: library/options.inc.php [hunk 2]
```diff
@@ -99,9 +99,9 @@ function generate_select_list($tag_name, $list_id, $currvalue, $title, $empty_na
 		$tag_id_esc = attr($tag_id);
 	}
 	
-	if ($multiple) {
+	/*if ($multiple) {
 		$tag_id_esc = $tag_id_esc . "[]";
-	}
+	}*/
```
Comment (bradymiller) [inline library/options.inc.php]: Remove code rather than comment it out.

File: library/options.js.php [hunk 3]
```diff
@@ -60,6 +60,7 @@ function updateAgeString(fieldid, asof, format) {
 //
 var cskerror = false; // to avoid repeating error messages
 function checkSkipConditions() {
+    debugger;
```
Comment (bradymiller) [inline library/options.js.php]: what is this?

**Comment by matrix-olegsv** [comment]
@bradymiller I guess it's ready for another review?

**Comment by bradymiller** [comment]
This breaks the Race entry in demographics which is a multiselect. Also see the following warning when save the patient demographics:
07-Jul-2016 22:02:35 America/Los_Angeles] PHP Warning:  Invalid argument supplied for foreach() in /var/www/openemr/library/options.inc.php on line 2755

**Comment by matrix-olegsv** [comment]
@bradymiller We have submitted an improved solution, please check.

**Comment by matrix-olegsv** [comment]
@bradymiller do you have any more comments on this or its accepted?

**Comment by bradymiller** [comment]
hi,
thanks for the reminder. not sure why i didn't see the prior message. will test/review it very soon.
-brady

**Comment by bradymiller** [comment]
Hi,
I ensured multiselect selector is working on the forms (I did not test the condition logic and I assume you have tested this). I just committed this to the main codebase. thanks for the fix!
-brady

**Pull request closed without merging**